# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/gdb/mozilla/unwind.py
# Commit: cda8bed19391
# Full Hash: cda8bed193914e4e6bedf875aecd9abbcc74606e
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2024-08-02 15:37:12
# Description:
#   Bug 1910880 - Use a real JIT exit frame for Wasm JitEntry stub. r=rhunt
#   
#   JS JIT callers can call Wasm code either directly from Ion or through the JitEntry stub.
#   In the JitEntry case, we didn't have a real JS JIT exit frame. To make JS frame iteration
#   still work in this case, a special `JSJitToWasm` frame type was used to transition from Wasm
# ==============================================================================

diff -r 9b92adc2e91f -r cda8bed19391 js/src/gdb/mozilla/unwind.py
--- a/js/src/gdb/mozilla/unwind.py	Fri Aug 02 06:59:32 2024 +0000
+++ b/js/src/gdb/mozilla/unwind.py	Fri Aug 02 07:14:08 2024 +0000
@@ -49,7 +49,6 @@
     "FrameType::BaselineStub": "BaselineStubFrameLayout",
     "FrameType::CppToJSJit": "JitFrameLayout",
     "FrameType::WasmToJSJit": "JitFrameLayout",
-    "FrameType::JSJitToWasm": "JitFrameLayout",
     "FrameType::Rectifier": "RectifierFrameLayout",
     "FrameType::IonAccessorIC": "IonAccessorICFrameLayout",
     "FrameType::IonICCall": "IonICCallFrameLayout",